README.md

```diff
@@ -1,125 +1,5 @@
-# USWDS Gulp pipeline for copying assets and compiling Sass
+# [Deprecated] USWDS Gulp pipeline for copying assets and compiling Sass
 
-A simple [Gulp 4.0](https://gulpjs.com/) workflow for transforming USWDS Sass into browser-readable CSS.
+⚠️ This project is no longer under active development. Please use [USWDS/Compile](https://www.npmjs.com/package/@uswds/compile) instead.
 
-**Note:** You do _not_ need to clone this repo into your project. Follow the instructions below.
```

Comment:
What do you think about keeping the instructions in the readme, and just adding a deprecation note at the top? I feel like it would be helpful for current users to have a reference if they are not yet able to make the switch.

Reply:
My vote is to keep the instructions there, but add more attention by adding something like a  🚩 emoji and making the text bigger and/or bolder. Happy to defer though.

Reply:
@amyleadem thanks for the suggestions. I've moved the text to the top and added a header that says deprecated.